library/core/registry/src/main/java/org/quiltmc/qsl/registry/api/event/RegistryEntryContext.java

```diff
@@ -0,0 +1,39 @@
+package org.quiltmc.qsl.registry.api.event;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * Represents information about a registry entry.
+ *
+ * @param <V> The entry type used by the relevant {@link Registry}.
+ */
+public interface RegistryEntryContext<V> {
+	/**
+	 * Getter for the relevant registry.
+	 *
+	 * @return The relevant registry for this entry.
```

Comment:
```suggestion
	 * {@return the relevant registry for this entry}
```

I can't multiline suggest apparently, urgh.
but basically the `{@return}` replace the whole thing.